README.md

```diff
@@ -1,5 +1,14 @@
 # web-ui
 
+#Dependencies
+NPM- Installs with Node.js here: 
+https://nodejs.org/en/
```

Comment:
We should be a bit opinionated here and recommend the latest v8 series, as it's the current Long-Term Support version, and it's the version that our CI/CD tool is using to build all the deployed code.